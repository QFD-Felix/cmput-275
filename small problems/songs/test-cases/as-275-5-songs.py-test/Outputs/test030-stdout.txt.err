46 20 13 32 12 25 59 0 82 1 42 2 11 15 3 10 4 37 1 17 6 1 33 16 3 12 38 48 15 4 1 7 2 20 10 15 6 4 24 8 0 4 7 20 41 26 11 14 0 32 1 5 31 33 4 19 12 84 1 8 0 34 29 3 14 41 3 6 12 1 9 1 4 16 31 17 1 5 38 5 30 20 22 4 4 14 12 16 6 13 3 29 5 76 14 1 4 14 35 61 14 6 23 10 25 40 22 5 1 12 7 50 11 2 38 7 11 1 16 0 6 51 21 1 38 12 21 4 2 36 20 4 0 3 60 3 6 2 10 14 0 10 16 1 2 47 32 2 7 18 55 12 2 11 2 2 98 2 10 16 23 32 22 3 6 16 14 72 31 5 1 2 56 63 5 1 11 0 4 6 37 5 33 5 6 6 3 29 14 6 8 10 14 3 9 9 1 3 13 32 9 6 10 22 6 8 2 11 6 59 17 41 7 17 29 33 4 6 4 2 4 21 7 3 21 9 37 14 12 43 3 4 0 28 13 2 2 14 6 35 6 8 20 13 21 12 47 11 30 4 0 41 14 23 1 41 4 26 1 8 4 10 18 16 54 8 4 2 86 13 14 18 22 8 11 0 10 0 28 9 66 46 5 56 19 12 5 8 68 1 11 7 16 6 18 20 4 7 4 68 16 33 3 7 25 8 1 2 3 11 0 44 7 46 0 6 6 0 30 12 20 2 2 7 1 3 0 5 7 4 8 11 36 18 23 1 5 0 5 8 22 3 0 2 5 0 27 30 5 3 0 19 8 5 12 0 7 12 0 12 0 0 14 1 1 0 6 0 0 0 4 0 0 2 0 0 0 0 0
