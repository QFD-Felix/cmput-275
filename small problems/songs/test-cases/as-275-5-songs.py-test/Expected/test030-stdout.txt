49 23 14 35 13 28 1 1 60 85 1 44 2 13 4 4 12 6 39 12 0 1 17 6 1 33 16 3 12 38 2 15 4 1 7 2 20 10 15 6 4 24 8 0 4 7 20 42 27 11 14 0 33 1 5 32 34 4 20 13 19 1 9 0 35 30 4 15 42 0 3 6 12 1 9 1 4 16 31 17 1 5 38 5 30 20 22 4 4 14 12 16 6 13 3 29 5 76 14 1 4 14 35 61 14 6 23 10 25 40 22 5 1 12 7 50 11 2 38 7 11 1 16 0 6 51 21 1 38 12 21 4 2 36 20 4 0 3 60 3 6 2 10 14 0 0 9 1 2 47 32 2 7 18 55 12 2 11 2 2 99 2 0 9 23 32 22 3 6 16 14 73 31 5 1 2 57 64 5 0 0 0 4 6 38 5 33 4 6 6 3 29 0 6 8 10 14 3 9 9 1 3 4 33 9 6 10 8 6 8 2 12 6 60 18 42 8 4 30 34 5 7 13 4 2 4 21 7 3 21 0 8 14 12 43 3 4 0 28 13 2 2 14 6 35 6 8 20 13 21 12 47 11 30 4 0 41 14 23 1 41 4 26 1 8 3 10 18 16 0 8 4 2 87 13 14 18 22 8 11 0 10 0 28 9 67 46 5 56 19 12 5 8 69 1 11 7 16 6 18 20 4 7 4 69 16 33 3 7 25 8 1 2 3 9 0 45 7 47 0 6 6 0 31 1 20 2 2 7 1 3 0 4 7 4 8 11 0 19 11 1 5 0 5 9 23 3 0 3 6 0 12 27 30 5 3 0 19 8 5 12 0 7 12 0 12 0 0 6 1 1 0 6 0 0 7 4 0 0 2 0 0 0 0 0
